# HG changeset patch
# User Jeff A. Marr <jamarr@gleim.com
# Date 1374767456 14400
# Node ID 5a45403e1bff1307c83e7c4fcabec9d264da0a1a
# Parent  24c496bd44fcb3d81f0d66ae9037384cb72eeb56
[1] added documentation for wxHtmlWindow::GetDefaultHTMLCursor and wxHtmlWindow::SetDefaultHTMLCursor.

diff -r 24c496bd44fc -r 5a45403e1bff wxWidgets/interface/wx/html/htmlwin.h
--- a/wxWidgets/interface/wx/html/htmlwin.h	Sat Jul 20 13:04:39 2013 -0400
+++ b/wxWidgets/interface/wx/html/htmlwin.h	Thu Jul 25 11:50:56 2013 -0400
@@ -365,6 +365,25 @@
     virtual void WriteCustomization(wxConfigBase* cfg,
                                     wxString path = wxEmptyString);
 
+    /**
+        Retrieves the default cursor for a given HTMLCursor type. 
+
+        @param type
+            HTMLCursor type to retrieve.
+    */
+    static wxCursor GetDefaultHTMLCursor(HTMLCursor type);
+
+    /**
+        Sets the default cursor for a given HTMLCursor type. 
+        These cursors are used for all wxHtmlWindow objects by default, but can be overridden on a per-window basis.
+
+        @param type
+            HTMLCursor type to retrieve.
+        @param cursor
+            wxCursor to act as the new default cursor for the specified cursor type.
+    */
+    static void SetDefaultHTMLCursor(HTMLCursor type, const wxCursor& cursor);
+
 protected:
 
     /**
